run_robot_test.sh

```diff
@@ -114,10 +114,47 @@ case "$(uname -s)" in
         ;;
 esac
 
+## if we have yq installed
+if command -v yq &> /dev/null
+then
+    echo "INFO: we found a yq executable"
+
+    ## get the user, pass and API hostname for OpenShift
+    oc_user=$(yq  e '.OCP_ADMIN_USER.USERNAME' ${TEST_VARIABLES_FILE})
+    oc_pass=$(yq  e '.OCP_ADMIN_USER.PASSWORD' ${TEST_VARIABLES_FILE})
+    oc_host=$(yq  e '.OCP_API_URL' ${TEST_VARIABLES_FILE})
+
+    ## do an oc login here
+    oc login "${oc_host}" --username "${oc_user}" --password "${oc_pass}" --insecure-skip-tls-verify=true
```

Comment:
Could we handle the "oc login" in the OpenShift CLI library or in any other library file?

Reply:
okay, for now we can live with this, later if we find a better solution we can enhance it.